src/common/stacks-utils.ts

```diff
@@ -75,3 +75,8 @@ export function validateAddressChain(address: string, currentNetwork: Network) {
 }
 
 export const getTicker = (assetName: string) => assetName.slice(0, 3).toUpperCase();
+
+export const amountToDecimals = (value: number | string | BigNumber, decimals: number = 2) => {
```

Comment:
What's the relevance of 2 decimals, and why do we want this as a default vs being explicit every time?